### PR TITLE
Add Balan and Pollard site designs to Gutenboarding

### LINKED ISSUE #48381: Signup: Add two Starter Designs
We've created two new Starter Design sites that can be added to the Design Screen in signup.

1. https://pollardstarter.wordpress.com/
2. https://balanstarter.wordpress.com/

There are a few themes issues surfaced by these Starter Designs that would be good to get closed before these are added to the design screen in signup.

- https://github.com/Automattic/themes/issues/2896
- https://github.com/Automattic/themes/issues/2895
- https://github.com/Automattic/themes/issues/2918

Additionally, as these are "short" designs, they'll look their best with slightly shorter thumbnails. Something to consider for further improvements to this screen. cc @joanrho @rickybanister 

Speaking of, here's what they look like right now (with those issues present) …

image

cc @iamtakashi @alaczek 

### Recipe

The starter design "recipe" for both sites looks like the following.

- [Mayland Theme](https://maylanddemo.wordpress.com/).
- Reduce default content to one static homepage.
- Update block-based content in the homepage. (Expect the content here, and similar content to show up in the Editor as Patterns.)
- Update global fonts in the Editor.
- Set custom colour scheme and remove menu in the customizer.

These sites will not receive a blog page by default but just like a custom menu with additional pages that will be quite easy to add later.

We'll want to continue to make improvements to these Starter Designs as block-based themes are added to Signup.

### PR DESCRIPTION
**Pull request by andrewserong**
Apply diff D55054-code before testing creating a site so that you have access to the annotations and the template/demo preview for these two designs.

This PR adds in the new Balan and Pollard site designs (#48381)

#### Changes proposed in this Pull Request

* Add `balan` and `pollard` new site designs

__Note:__ the screenshots of these short designs don't work very well with the layout for picking a design in Gutenboarding, so another solution for the screenshots / layout for including these sorts of designs might be required. E.g. here's how they currently look:

| Balan | Pollard |
| --- | --- |
| image | image |

#### Testing instructions

* Apply D55054-code and sandbox the public-api
* Go to /new and at the design step you should see the new Balan and Pollard site designs
* You should be able to create a new site with each of these designs, and the newly created site should look like the designs

Fixes #48381

### CODE REVIEW

**Comment by ramonjd** [comment]
@p-jackson I added you as the assignee since you were on D55054-code

This looks great to me and I think it's shippable once D55054-code is deployed.

Tested the demos as well, e.g., 
https://public-api.wordpress.com/rest/v1/template/demo/mayland/balan/?language=en&site_title=wefgq543y5y8986uyrhtdgsf&font_headings=Rubik&font_base=Roboto

Screen Shot 2021-01-20 at 1 52 06 pm

As @andrewserong mentioned, the border looks a teensy bit strange:

Screen Shot 2021-01-20 at 1 49 02 pm

Wondering if we could remove the border, maybe show 4 cols to tighten it up? Not sure!

Screen Shot 2021-01-20 at 1 55 29 pm

@ianstewart Would you rather wait until it looks prettier? Or could we go ahead and add these designs to onboarding?

cc @simison in case we're adding any noise to running A/B tests for onboarding..

**Comment by simison** [comment]
>  Would you rather wait until it looks prettier? Or could we go ahead and add these designs to onboarding?

I think I'd prefer wait it looks better — this is an important first impression moment. How would it look if screenshots were landscape? Or do we need to support different height screenshots? Can anything (height wise) be adjusted in those templates? cc @ollierozdarz

**Comment by ollierozdarz** [comment]
> I think I'd prefer wait it looks better — this is an important first impression moment.

Agree, @simison. This is one of the more important moments at the start of the journey and if it appears like our templates are breaking (even though they aren't) it could cause some distrust.

**Comment by lsl** [comment]
Do these designs need to be displayed in a grid? It might be better to show 4 vertical columns of varying height designs. (add screenshot height variable to the design config.)

**Comment by ollierozdarz** [comment]
> Do these designs need to be displayed in a grid? It might be better to show 4 vertical columns of varying height designs. (add screenshot height variable to the design config.)

That's a good suggestion, @lsl. My gut feel is that this would still lend better to a structured grid over a masonry grid. Masonry grids can quickly feel messy and they're usually harder to digest, and this is one of those moments where we can easily lose a user.

**Comment by lsl** [comment]
Fair enough, might need to improve the themes for longer screens anyway looking at the tablet view:
Screenshot_2021-01-21 WordPress com(1)

This is how they look with [mshots](https://github.com/Automattic/wp-calypso/pull/43428) enabled:

Screenshot_2021-01-21 WordPress com

**Comment by lsl** [comment]
> Masonry grids can quickly feel messy and they're usually harder to digest, and this is one of those moments where we can easily lose a user.

Just saw this: pbAok1-1QJ-p2, I can see what you mean, I wonder if thicker borders or more whitespace would be enough but I did have trouble figuring out where I was supposed to move my eyes.

**Comment by ollierozdarz** [comment]
That could work, but might feel like a band-aid solution to the bigger issue? I'd be keen to see what landscape looks like (@simison's suggestion) for all of these designs, although I'm guessing it's not an easy comparison to spin up?

**Comment by lsl** [comment]
Landscape:
Screenshot_2021-01-21 WordPress com

Portrait:
Screenshot_2021-01-21 WordPress com(1)

Masonry (picked a few at random, used a mix of heights):
Screenshot_2021-01-21 WordPress com(2)

**Comment by simison** [comment]
Nice, thanks for the screenshots @lsl ! What do you folks think if we go with landscape, but consider making the screenshots bigger, like 3 or 2 columns?

FYI @joanrho

**Comment by joanrho** [comment]
Thanks for the ping, @simison!

The landscape direction definitely feels more structured than the masonry grid, as it's easier to scan across and down the page within the clearly defined grid of thumbs. The only limitation that comes to mind is that less of the design is visible since we currently don't offer previews of the full page design. 

If we can include a "Preview" option for users to view the full page design before making a selection, I think that would be the best option. We've explored this idea before in past mocks—something like an overlay on hover with Preview/Select buttons where Preview calls up a modal displaying a larger, scrollable full-page design with device preview options and a Select button to continue or Back/close to return to the grid of options.

On my end, Ganon is also working on the New Page Layout Modal (see pbAok1-1QJ), so it shouldn't be too difficult to adjust that design from the masonry layout to the landscape direction if we choose to go that route and align both pages.

**Comment by lsl** [comment]
> If we can include a "Preview" option for users to view the full page design before making a selection

@deBhal had something in the works for a scrolling gif on mouseover that could work for this.

Squarish(90% h) preview might be enough here too:
Screenshot_2021-01-22 WordPress com(2)

Or Squarish(80% h) / 3 columns: (we currently use this for smaller screens, 4 is only shown on "gigantic" screens)
Screenshot_2021-01-22 WordPress com(3)

(new designs might need some extra help to look right in this odd browser shape)

**Comment by ollierozdarz** [comment]
Nice sentiment, @joanrho! Completely agree, Landscape ratio with a preview modal would be our dream state here (show more designs at once, make the look more like a cover as opposed to every detail, give a big preview for folks who want to see more details). This is basically my initial theory of how the Theme showcase can/should look. 

In the meantime, keeping in mind this task might be more time sensitive, what are your thoughts on running with Landscape cards without a preview, @simison? This could be an interesting side test to see if Landscape ratio influences completion of the onboarding flow more or less?

**Comment by lsl** [comment]
Landscape change happening in #49170 - added a hacky :hover effect that sort of works. It has some side effects and a bunch of tweaking is needed. Probably also need something extra to support Balan/Pollard as well.

landscape-hover

Not sure if this is what we want but figured it would be a good way to get a feel for how something like this might look/work.

**Comment by ollierozdarz** [comment]
This is really great work, @lsl. I'd be inclined to up the hover speed (maybe just double it), but this is lots of value (for desktop anyway).

**Comment by lsl** [comment]
Rebased to include landscape/#49170 changes, will circle back and disable the hover scroll for these two designs only to avoid this:

Peek 2021-01-28 16-43

**Comment by simison** [comment]
@ianstewart @ollierozdarz thoughts on if to release this as-is or if to disable the animation for these designs?

**Comment by ollierozdarz** [comment]
Jumping in before @ianstewart here 🙂 — but I'd personally disable it for these designs so that it indicates that they're more of a fixed size.

**Comment by lsl** [comment]
Static designs can be flagged in the design config/json file and they'll get a different css rule.

static

File: client/landing/gutenboarding/stores/onboard/types.ts [hunk 1]
```diff
@@ -41,6 +41,7 @@ export interface Design {
 	src: string;
 	template: string;
 	theme: string;
+	preview?: string;
```
Comment (p-jackson) [inline client/landing/gutenboarding/stores/onboard/types.ts]: @lsl not a blocker, just thinking about whether the `preview` type could be annotated like this instead:

```typescript
preview?: 'static';
```

Just so when we want to add another one of these we don't have to dig around in the code to remember what the special value of the flag is supposed to be.

**Review by p-jackson — APPROVED** [review]
LGTM

**Comment by lsl** [comment]
Blocked while a couple of theme issues are sorted out: https://github.com/Automattic/wp-calypso/issues/48381#issuecomment-772225125

**Comment by simison** [comment]
Theme fixes are now deployed to .com, and colors work:

image

**Review by simison — APPROVED** [review]
Tests well, tried creating both sites and testing preview as well editor.

**Comment by simison** [comment]
I can roll this out since I'm already looking at it and it's been also tested by the team. Hope you don't mind! :-)

**Pull request merged**